model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -95,31 +104,35 @@ def set_last_results(self, measurements):
 
     def _create_concurrency_list(self, cli_config, model_parameters):
         if model_parameters['concurrency']:
-            return model_parameters['concurrency']
+            return sorted(model_parameters['concurrency'])
```

Comment:
sorted to guarantee we are always strictly increasing concurrency

However... are we even supposed to early exit if the user specified a fixed list? Shouldn't we only be exiting on the auto-search?